tox.ini

```diff
@@ -15,7 +15,11 @@ deps =
   pep8-naming
   wheel
 commands = nosetests []
-passenv = HOME
+passenv =
+    CURL_CA_BUNDLE
+    HOME
+    REQUESTS_CA_BUNDLE
+    SSL_CERT_FILE
```

Comment:
Before someone asks: these are needed if SSL proxies, othewise testing would fail with failure to validate certificates.

Reply:
Instead of a comment in PR, the best practice is to put a comment in the file itself.